keanu-python/keanu/tensor.py

```diff
@@ -51,11 +50,26 @@ def get_tensor_type(self) -> type:
     def __get_tensor_from_ndarray(ndarray: numpy_types) -> JavaObject:
 
         ctor = Tensor.__infer_tensor_ctor_from_ndarray(ndarray)
-        values = k.to_java_array(ndarray.flatten().tolist())
+        values = Tensor.__get_java_array_from_ndarray(ndarray)
         shape = k.to_java_long_array(ndarray.shape)
 
         return ctor(values, shape)
 
+    @staticmethod
+    def __get_java_array_from_ndarray(ndarray: numpy_types) -> Union[JavaObject, bytes]:
```

Comment:
is there a way to make it always return `bytes` and never `JavaObject`?

Reply:
I've now done it so it only returns JavaObject, which I think still captures the spirit of this comment?

Reply:
👍